src/console/plugin.ts

```diff
@@ -200,9 +200,9 @@ function activateConsole(app: Application, services: ServiceManager, rendermime:
 
   let newSubmenu = new Menu ([
     new MenuItem ({
-      text: 'Python 3 Console',
+      text: services.kernelspecs.default,
```

Comment:
Ahh ok! Thanks! That is just what I needed. Will update this PR so that it can have the ability to open any X Console given that kernel is installed!